app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt

```diff
@@ -41,48 +41,52 @@ import com.infomaniak.swisstransfer.ui.utils.ScreenWrapperUtils
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
 fun TransfersScreenWrapper(direction: TransferDirection) {
-    TwoPaneScaffold<String>(
+    TwoPaneScaffold<Pair<TransferDirection, String>>(
         listPane = { ListPane(direction, navigator = this) },
         detailPane = { DetailPane(navigator = this) },
     )
 }
 
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
-private fun ListPane(direction: TransferDirection, navigator: ThreePaneScaffoldNavigator<String>) {
+private fun ListPane(direction: TransferDirection, navigator: ThreePaneScaffoldNavigator<Pair<TransferDirection, String>>) {
```

Comment:
We could make a dataClass for this Pair as it's used in a lot of places

Reply:
We can't store a data class in a destination's content :(